hudi-utilities/src/main/java/org/apache/hudi/utilities/HoodieMetadataTableValidator.java

```diff
@@ -283,6 +284,9 @@ public static class Config implements Serializable {
     @Parameter(names = {"--validate-all-file-groups"}, description = "Validate all file groups, and all file slices within file groups.", required = false)
     public boolean validateAllFileGroups = false;
 
+    @Parameter(names = {"--validate-last-n-file-slices"}, description = "Validate just the last N file slices for all file groups", required = false)
+    public String validateLastNFileSlices = null;
```

Comment:
Just to clarify usage, maybe we can say - `Validate just the last N file slices for all file groups. Specify N.` Also, why not declare as Integer, instad of doing String and parsing as int later?